src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java

```diff
@@ -119,6 +119,16 @@ private void initialize() {
                         }
                     }).executeWith(Globals.TASK_EXECUTOR);
 
+                    /*
+                    inserted the if-statement here, since a Platforn.runLater() call did not work.
+                    also tried to move it to the end of the initialize method, but it did not select the entry.
+                    */
+                    if (entriesListView.getItems().size() == 1) {
+                        selectAllNewEntries();
+                      }
+  
+                      return container;
```

Comment:
Now you have two return statements 🦊

Reply:
I am so sorry for all the inconveniences. I'll correct it immediately.